app/move/views/create/move-not-supported.njk

```diff
@@ -0,0 +1,14 @@
+{% extends "layouts/action-prevented.njk" %}
+
+{% set actionPreventedTitle %}
```

Comment:
A small note, but these can also be defined without the need for a block:

```nunjucks
{% set actionPreventedTitle = t("validation::move_not_supported.heading", {
  name: person.fullname
}) %}
```

The benefit of this is that you won't be injecting unnecessary whitespace into the final markup.